invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -262,6 +262,50 @@ <h2 id="files-heading">{{ _('Files') }}</h2>
               {%- endif %}
             {%- endblock record_files -%}
 
+            {# Files #}
+            {%- block record_media_files -%}
+              {# record has files BUT passed files are empty. This happens when we display are request. #}
+              {%- if record.media_files.enabled -%}
+                <section id="record-media-files" class="rel-mt-2" aria-label="{{ _('Media files') }}">
+                  <h2 id="files-heading">{{ _('Media files') }}</h2>
+                  {%- if permissions.can_view -%}
+                    {# record has files AND user can see files #}
+                    {%- set media_files = media_files|order_entries %}
+
+{#                    {%- if media_files|has_previewable_files -%}#}
```

Comment:
I assume we don't want the preview here, to be confirmed (I will remove the commented out code)